dbld/README.md

```diff
@@ -28,33 +28,35 @@ The source code and build products are mounted externally in the following locat
 
 ## Examples
 
-### Building syslog-ng from source using Ubuntu Xenial
-> Assume that we have cloned syslog-ng's source into the `$HOME/syslog-ng` directory.
-
-The following commands starts a container mounted with the source:
+### Building syslog-ng from tarball using a supported distribution (e. g. Ubuntu Focal)
```

Comment:
As the tarball contains the `dbld`, I would suggest to describe this part as if the user starting from tarball.
Something like:
1. download syslog-ng tarball
2. untar
3. dbld/rules shell-ubuntu-focal
4. confiugre && make

Reply:
I would propose something different.

If a user would like to build on a platform, where bison is old (< 3.7.6) we should promote `dbld/rules tarball` target instead of manually fetching the tarball from GitHub.
This README.md is for dbld. For building syslog-ng from tarball, we have the main [README.md](https://github.com/syslog-ng/syslog-ng/blob/master/README.md) file

Reply:
thanks for the change!